twitter_clone/templates/layout.html

```diff
@@ -0,0 +1,59 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Twitter clone</title>
+
+        <!-- styles -->
+        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
+        <link rel="stylesheet" href="../../static/css/styles.css">
+    </head>
+    <body>
+        <div class="container body-container">
+
+            <nav class="navbar navbar-default navbar-fixed-top">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/">Twitter clone</a>
+    </div>
+    <div id="navbar" class="navbar-collapse collapse">
+      <ul class="nav navbar-nav navbar-right">
+        {% if session['username'] %}
+            <li><a href="/profile"><i class="fa fa-user" aria-hidden="true"></i>{{ session['username'] }}</a></li>
+            <li><a href="/logout?next=%2Fmartinzugnoni"><i class="fa fa-sign-out" aria-hidden="true"></i> Log out</a></li>

```

Comment:
Yes, actually should have just changed the parameter so the next directs at the current page.